utils/src/style-spec.ts

```diff
@@ -173,7 +171,7 @@ export {
 
     ColorType,
     interpolates,
-    v8,
+    latest as v8,
```

Comment:
I think dts-bundle-generator has problems with exporting `something as somthingelse`...
I would reccomend simply importing it using `import v8 from './style-spec/latest'`.